example/dice/otel.go

```diff
@@ -83,12 +78,8 @@ func setupOTelSDK(ctx context.Context, serviceName, serviceVersion string) (shut
 	return
 }
 
-func newResource(serviceName, serviceVersion string) (*resource.Resource, error) {
-	return resource.Merge(resource.Default(),
-		resource.NewWithAttributes(semconv.SchemaURL,
-			semconv.ServiceName(serviceName),
-			semconv.ServiceVersion(serviceVersion),
-		))
+func newResource() *resource.Resource {
```

Comment:
Isn't this used by default?  Should we remove resource setting from the example entirely?

Reply:
Sorry I got you wrong. I see you mean completely remove setting of the resource... Makes sense. I will do that unless anyone disagrees.

@MrAlias @dmathieu are you OK with removing the code that sets the resource?

Reply:
> @MrAlias @dmathieu Are you OK with removing the code that sets the resource?

SGTM

Reply:
Addressed eb1decfbc677d43d84daa017ba0eeece56c6962f